R/dist-fitting.R

```diff
@@ -8,8 +8,8 @@
 #'    fit_best_single(rnorm(100), "normal")
 #'    # fit_best_single(rtriangle(100), "triangle")
 fit_best_single <- function(data, distribution){
-  x <- list(data = data, distribution = distribution)
-  class(x) <- c(class(x), distribution)
+  x <- list(data = data, distribution = convert_dist(distribution))
+  class(x) <- c(class(x), convert_dist(distribution))
```

Comment:
This won't affect anything on the UI right? Remember that we do a mapping there as well.